src/utilities/imageLoader.js

```diff
@@ -0,0 +1,15 @@
+export const dq = [
+  require('../assets/img/posts/dq/dq_1.jpg'),
+  require('../assets/img/posts/dq/dq_2.jpg'),
+  require('../assets/img/posts/dq/dq_3.jpg'),
+  require('../assets/img/posts/dq/dq_4.jpg'),
+  require('../assets/img/posts/dq/dq_5.jpg')
```

Comment:
Do you have a plan for refactoring this? We can't scale this out.

Reply:
This is being refactored to storing images on S3.